lib/plugins/requirements-txt.js

```diff
@@ -0,0 +1,28 @@
+import { REQUIREMENTS_TXT } from '../../packages/helper-grammar-regex-collection/index.js';
+import insertLink from '../insert-link';
+import preset from '../pattern-preset';
+import liveResolverQuery from '../resolver/live-resolver-query.js';
+
+export default class RequirementsTxt {
+
+  static resolve({ target }) {
+    return [
+      liveResolverQuery({
+        target,
+        type: 'pypi',
+      }),
+    ];
+  }
+
+  getPattern() {
+    return preset('RequirementsTxt');
+  }
+
+  parseBlob(blob) {
+    insertLink(blob.el, REQUIREMENTS_TXT, {
+      pluginName: this.constructor.name,
+      target: '$1',
+      path: blob.path,
+    });
+  }
+}
```

Comment:
Yes, but I think it makes more sense to keep it separate, since the resolution logic is different. In particular, we don't need to do any local file, init, or api detection (all the stuff [here](https://github.com/OctoLinker/browser-extension/blob/7c785a581facd284519c70f2abad18b11855d8ce/lib/plugins/python.js#L10-L26)).

Additionally, since the python resolver [prefers](https://github.com/OctoLinker/browser-extension/blob/7c785a581facd284519c70f2abad18b11855d8ce/lib/plugins/python.js#L29) Python API docs over userland packages, we could end up resolving a requirements.txt entry of (for example) `argparse` to https://docs.python.org/3/library/argparse.html instead of https://github.com/ThomasWaldmann/argparse/. This is the opposite of the problem described at https://github.com/OctoLinker/browser-extension/pull/210